iam/terraform/environment-specific/bin/deploy-app.sh

```diff
@@ -5,6 +5,7 @@ ENVIRONMENT=$1
 # Getting the environment-specific deployment settings and injecting them into the shell environment
 export ENV=${ENVIRONMENT}
 pushd ../../../../
+# shellcheck disable=SC1091
```

Comment:
What is this disabling a check for?

Reply:
shellcheck doesn't like sourcing environment variables from external files.  There is a flag you can use to run in shellcheck like `shellcheck -x` to suppress this error, but the staged plugin doesn't allow custom flag to be passed in.